commit ce9b2f274bf81a44d16b9a01396eafdbf5887101 (refs/changes/84/5084/1)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2018-11-21T16:29:01+00:00 (11 months ago)
    
    MANTA-4024 madtom's nodunit dep no longer installable with bundled node version

diff --git a/Makefile b/Makefile
index 27176bc..a84604e 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2017, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 #
@@ -84,7 +84,7 @@ CLEAN_FILES += $(NODEUNIT) ./node_modules/nodeunit
 .PHONY: test
 test: $(NODEUNIT)
 	mkdir -p ./tmp
-	find test/ -name '*.test.js' | xargs -t -L1 ctrun -o noorphan node
+	find test/ -name '*.test.js' | xargs -t -L1 ctrun -o noorphan $(NODE_EXEC)
 
 .PHONY: scripts
 scripts: deps/manta-scripts/.git
diff --git a/package.json b/package.json
index 3992d81..4024aa4 100644
--- a/package.json
+++ b/package.json
@@ -17,7 +17,8 @@
                 "vasync": "1.5.0"
         },
         "devDependencies": {
-                "nodeunit": "0.7.4"
+                "nodeunit": "0.10.1",
+                "debug": "3.1.0"
         },
         "engines": {
                 "node": "0.8.18"
